src/cross_section/include/cross_section.h

```diff
@@ -36,7 +36,9 @@ class CrossSection {
 
   // Shapes
   static CrossSection Square(glm::vec2 dims, bool center = false);
-  static CrossSection Circle(float radius, int circularSegments);
+  static CrossSection Circle(float radius, int circularSegments = 0);
+  static CrossSection Ellipse(float radiusX, float radiusY,
```

Comment:
In keeping with the Manifold/OpenSCAD API, I think we can skip this, since it's just `Circle().Scale()`.

Reply:
Sure, makes sense.